rabbitmq/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # CHANGELOG - rabbitmq
 
 
+1.4.0 / 2017-09-04
```

Comment:
Since the integration has not been release yet (I set the milestone to 5.18), could you change the date to `Unreleased` ?

Reply:
Done